pkg/apis/eksctl.io/v1alpha5/validation.go

```diff
@@ -340,13 +340,15 @@ func (c *ClusterConfig) validateKubernetesNetworkConfig() error {
 			}
 		}
 
-		if c.KubernetesNetworkConfig.IPFamily != IPV4Family && c.KubernetesNetworkConfig.IPFamily != IPV6Family {
-			return fmt.Errorf("invalid value %s for ipFamily; allowed are %s and %s", c.KubernetesNetworkConfig.IPFamily, IPV4Family, IPV6Family)
-		}
+		switch c.KubernetesNetworkConfig.IPFamily {
```

Comment:
I'm not sure I'm following this change. What was the issue here?

Reply:
It was throwing a validation error when ipFamily was empty. `case IPV4Family, "":` ensures empty values are treated as IPv4.

Reply:
Ah it was missing a default case of empty being ipv4. 🤦 gotcha. Can you please also add a validation test for this?

Reply:
Done.